run.py

```diff
@@ -532,6 +551,7 @@ def run(command, env={}, log_filename={}):
                                                project.anat_config_file,
                                                project.dmri_config_file,
                                                None,
+                                               project.eeg_config_file if run_eeg else None,
```

Comment:
Very good! Actually I should review code and make the same for `run_dmri` and `run_fmri`